### PR TITLE
nested-datatypes Blog Post

### PR DESCRIPTION
**Pull request by hgoldstein95**
I think we're just about good to go with this post. I had to do a little bit of manual clean-up after pandoc, so I'm not sure that the formatting is 100% perfect.

If possible I'd like to post this this afternoon, so if someone can give this a quick read before then that would be great.

### CODE REVIEW

File: blog/2020-12-04-nested-datatypes/assets/Nested.lhs
Comment (sweirich) [inline blog/2020-12-04-nested-datatypes/assets/Nested.lhs:623]: This link is unbound. Should be to https://github.com/sweirich/dth/blob/master/nested-datatypes/DFinger.lhs

**Review by sweirich — CHANGES_REQUESTED** [review]

File: blog/2020-12-04-nested-datatypes/2020-12-04-nested-datatypes.md
Comment (sweirich) [inline blog/2020-12-04-nested-datatypes/2020-12-04-nested-datatypes.md:743]: Better to link to the dth repo itself. That file has a link to other files in the same repo (along with a cabal file etc).

**Review by sweirich — CHANGES_REQUESTED** [review]

File: blog/2020-12-04-nested-datatypes/2020-12-04-nested-datatypes.md
Comment (Lysxia) [inline blog/2020-12-04-nested-datatypes/2020-12-04-nested-datatypes.md]:
```suggestion
This blog post was generated from a [literate Haskell file](./assets/Nested.lhs).
```

File: blog/2020-12-04-nested-datatypes/2020-12-04-nested-datatypes.md
Comment (Lysxia) [inline blog/2020-12-04-nested-datatypes/2020-12-04-nested-datatypes.md]: This and the URL in footnote 6 aren't clickable.

File: blog/2020-12-04-nested-datatypes/2020-12-04-nested-datatypes.md
Comment (hgoldstein95) [reply blog/2020-12-04-nested-datatypes/2020-12-04-nested-datatypes.md]: Done.

**Review by Lysxia — COMMENTED** [review]
I did a pass to check formatting. No issues there. I left two comments.

**Comment by hgoldstein95** [comment]
I think we're good, but let's hold off until Monday morning to post. I don't think anyone is going to want something new to do at 3pm on a Friday.

**Comment by hgoldstein95** [comment]
@dunnl How do we feel about merging this?

**Comment by dunnl** [comment]
@hgoldstein95 We feel great about it! I gave it a quick scan and don't have anything to add. I'll merge and if you want to deploy it just give me the word.

**Pull request merged**
